compiler-core/src/error.rs

```diff
@@ -2292,11 +2292,9 @@ implementation but the function name `{function}` is not valid."
                 TypeError::UnsupportedTarget {
                     location,
                     target: current_target,
-                    kind,
                 } => {
                     let text = wrap_format!(
-                        "This {} doesn't have an implementation for the {} target.",
-                        kind,
+                        "This value doesn't have an implementation for the {} target.",
```

Comment:
I opted to remove the `kind` (`function`/`constant`) because it felt a bit awkward reading the error message